applications/FluidDynamicsApplication/custom_python/add_custom_response_functions_to_python.cpp

```diff
@@ -0,0 +1,58 @@
+// External includes
+#include "pybind11/pybind11.h"
+
+// Project includes
+#include "response_functions/adjoint_response_function.h"
+
+// Application includes
+#include "custom_response_functions/drag_response_function.h"
+#include "custom_python/add_custom_response_functions_to_python.h"
+
+namespace Kratos
+{
+
+template<unsigned TDim>
+class PyDragResponseFunction : public DragResponseFunction<TDim>
+{
+    public:
+        using DragResponseFunction<TDim>::DragResponseFunction;
+        void Initialize() override {
+            PYBIND11_OVERLOAD(void, DragResponseFunction<TDim>, Initialize, );
+        }
+        void InitializeSolutionStep() override {
+            PYBIND11_OVERLOAD(void, DragResponseFunction<TDim>, InitializeSolutionStep, );
+        }
+        void FinalizeSolutionStep() override {
+            PYBIND11_OVERLOAD(void, DragResponseFunction<TDim>, FinalizeSolutionStep, );
+        }
+        double CalculateValue() override {
+            PYBIND11_OVERLOAD_PURE(double, DragResponseFunction<TDim>, CalculateValue, );
+        }
+};
+
+
+namespace Python
+{
+using namespace pybind11;
```

Comment:
please do not do this => use `namespace py = pbind11`
Otherwise there are some problems in windows => long term plan is to change it everywhere but I haven't had the time yet
I already updated e.g. the MappingApp and the Application-Generator, you can take a look there